app/Actions/UserList/GetUserListsCollectionAction.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace Hedonist\Actions\UserList;
+
+use Hedonist\Repositories\UserList\UserListRepository;
+
+class GetUserListsCollectionAction
+{
+    private $userListsRepository;
+
+    public function __construct(UserListRepository $userListsRepository)
+    {
+        $this->userListsRepository = $userListsRepository;
+    }
+
+    public function execute(int $userId): GetUserListsCollectionResponse
```

Comment:
execute should accept GetUserListsCollectionRequest class which will have $userId prop